tfx/examples/cifar10/README.md

```diff
@@ -0,0 +1,67 @@
+
+# CIFAR-10 Transfer Learning and MLKit integration Example
+This example illustrates how to use Transfer Learning for image classification with TFX, and use trained model to do object detection with [MLKit](https://developers.google.com/ml-kit)
+
+## Instruction
+
+Create a Python 3 virtual environment for this example and activate the
+`virtualenv`:
+
+```
+virtualenv -p python3.7 cifar10
+source ./cifar10/bin/activate
+```
+
+Then, clone the tfx repo and copy cifar10/ folder to home directory:
+
+```
+git clone https://github.com/tensorflow/tfx ~/tfx-source && pushd ~/tfx-source
+cp -r ~/tfx-source/tfx/examples/cifar10 ~/
+```
+
+Next, install the dependencies required by the CIFAR-10 example (appropriate
+version of TF2 will be installed automatically).
+
+```
+pip install -e cifar10/
+# The following is needed until tensorflow-model-analysis 0.23.0 is released
+pip uinstall tensorflow-model-analysis
```

Comment:
pip uinstall tensorflow-model-analysis
Should be:
pip uninstall tensorflow-model-analysis

Reply:
Thanks! I'll fix it